spec/features/user_session_timeout_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'features/support/sign_up_helpers'
+require 'features/support/not_signed_in'
+require 'timecop'
+
+describe 'with devise timeoutable enabled' do
```

Comment:
I wouldn't describe the technology used in the top-level part of these specs - I'd talk in an imperative way like I do with git commits , e.g. `describe 'logout users after long inactivity'`. 

Useful way to remember this is
> If I get this test passing, the application can/will now _do the described thing_

> If I get this test passing, the application will now _logout users after long inactivity_

i'd use the same (or very similar) description for the file name, too.

In general, if I see a `*with* some context` block, I expect to see that context being set up right there in the test - e.g.
```ruby
describe 'with a user' do
  let(:user) { create(:user) }
  # some expectations
end
```